### DISCUSSION TITLE
How to limit user connection via xrdp

### DISCUSSION THREAD

**Discussion by InoxTrm** [neutrinolabs/xrdp#2540] [Q&A]
How to limit user connection via xrdp

**Comment by metalefty** [comment]
It depends on the meaning of limit.

**Reply by InoxTrm** [reply]
Users are getting multiple VNC session or port. I want to limit it to one.

**Reply by matt335672** [reply]
@InoxTrm - some questions for you:-
- Version of xrdp?
- Platform?
- Backend in use (Xvnc/Xorg) ?
